openpype/modules/deadline/abstract_submit_deadline.py

```diff
@@ -627,7 +628,14 @@ def submit(self, payload):
             self.log.debug(payload)
             raise RuntimeError(response.text)
 
-        result = response.json()
+        try:
+            result = response.json()
+        except json.decoder.JSONDecodeError:
```

Comment:
Exception `JSONDecodeError` in `json.decoder` is not available in Python 2 DCCs. Put this to the header and use `JSONDecodeError` here.
```
JSONDecodeError = getattr(json.decoder, "JSONDecodeError", ValueError)
```